README.md

```diff
@@ -1,3 +1,3 @@
-# pb026393.github.io
-
-SMART ON FHIR STARTER APP
+# pb026393.github.io
+
+SMART ON FHIR STARTER APP

```

Comment:
Let's add a link from here to the documentation page: https://parthivbhagat.github.io/slate/index.html.  Eventually, this will live somewhere else.